MvvmCross/ViewModels/IMvxApplication.cs

```diff
@@ -12,8 +12,13 @@ public interface IMvxApplication : IMvxViewModelLocatorCollection
 
         void Initialize();
 
-        void Startup(object hint);
+        object Startup(object hint);
```

Comment:
@martijn00 I'm not sure why we're keeping the object parameter and return type since this can be done by using MvxApplication<THint> with THint set to object

Reply:
I just wanted to keep this to not break to many things. But it doesn't make a lot of sense. Developers could even just put `object` as type on the typed Startup.